sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/processor/CompactBlockProcessor.kt

```diff
@@ -329,7 +339,7 @@ class CompactBlockProcessor internal constructor(
         stop()
     }
 
-    suspend fun checkErrorResult(failedHeight: BlockHeight?) {
+    suspend fun checkContinuityErrorResult(failedHeight: BlockHeight?) {
         if (consecutiveChainErrors.get() >= RETRIES) {
```

Comment:
Conversely, we _don't_ want to abort after a set number of `ContinuityError` occurrences; reorgs can potentially occur frequently (e.g. during network weather), and the wallet needs to follow them no matter how many occur.

It feels like `checkContinuityErrorResult` should be a separate handler (or just call `handleChainError` directly, perhaps renamed) instead of repurposing `checkErrorResult`.

Reply:
Okay, that is a new requirement for the rewind logic in comparison with the existing implementation. I've pushed a change that keeps trying to reorg with a changed rewind height (i.e. calls `handleChainError` directly). Thanks for pointing this out!